omarchy-plugin/lib/pomo_link/paths.py

```
@@ -0,0 +1,54 @@
+def home():
+    return os.environ.get("HOME") or os.path.expanduser("~")
+
+
+def socket_path():
+    override = os.environ.get("POMO_LINK_SOCKET")
+    if override:
+        return override
+    runtime = os.environ.get("XDG_RUNTIME_DIR")
+    if runtime:
+        return os.path.join(runtime, "pomo", "pomo-link.sock")
+    return os.path.join("/tmp", "pomo-%s" % os.getuid(), "pomo-link.sock")
```

Ruff (0.16.3)

[error] Probable insecure usage of temporary file or directory: "/tmp"

(S108)